package.json

```diff
@@ -33,9 +33,10 @@
     "babel": "^5.5.6",
     "css-loader": "^0.14.5",
     "node-sass": "^3.1.2",
+    "rng": "^0.2.2",
     "sass-loader": "^1.0.2",
+    "seedrandom": "^2.4.0",
```

Comment:
seedrandom was already in there. i think you should be able to revert this file entirely